using-mobile-extensions/adobe-target/README.md

```diff
@@ -473,6 +474,11 @@ If you need to update SDK configuration, programmatically, use the following inf
 | target.clientcode | Client code for your account. |
 | target.timeout | Time, in seconds, to wait for a response from Target servers before timing out. |
 | target.environmentId | Environment ID you want to use, if this is left blank, the default production environment will be used. |
+| target.propertyToken | at_property token value which is generated from Target UI. If this is left blank, then no token will be sent in target network calls. |
+
+{% hint style="warning" %}
+It is recommended to use Launch config to pass property token instead of passing it inside mbox parameter. If property token is passed inside both launch configuration and mbox parameter, then the token entered in mbox parameter will be discarded.
```

Comment:
Please update the note so that it reads as follows:

"We recommend that you use an Experience Platform Launch config to pass the property token instead of passing it in an mbox parameter. If the property token is passed in an Experience Platform Launch configuration and an mbox parameter, the token that was entered in the mbox parameter is discarded."

Reply:
changed to 
"We recommend that you use Experience Platform Launch config to pass the property token instead of passing it in as a mbox parameter. If the property token is passed in Experience Platform Launch configuration and as a mbox parameter, the token that was entered in the mbox parameter is discarded."